sendwp/handler.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+add_action( 'wp_ajax_caldera_forms_sendwp_remote_install', 'wp_ajax_caldera_forms_sendwp_remote_install_handler' );
+function wp_ajax_caldera_forms_sendwp_remote_install_handler () {
+    $all_plugins = get_plugins();
+    $is_sendwp_installed = false;
```

Comment:
SECURITY There is no nonce check or authorization check. Please using existing APIs for capability checks and nonce generation.